### PR TITLE
Add advisory for segfault bug in simd-json.rs

### PR DESCRIPTION
**Pull request by Licenser**

### CODE REVIEW

**Comment by tarcieri** [comment]
@Licenser can you rebase? Looks like you missed some recent commits to `master`

File: crates/simd-json/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[advisory]
+id = "RUSTSEC-0000-0000"
+
+package = "simd-json"
+
+# Disclosure date of the advisory as an RFC 3339 date (mandatory)
```
Comment (tarcieri) [inline crates/simd-json/RUSTSEC-0000-0000.toml]: Please remove the comments. Thank you!

Comment (Licenser) [reply crates/simd-json/RUSTSEC-0000-0000.toml]: damn it sorry, I missed that one

**Comment by tarcieri** [comment]
Assigned `RUSTSEC-2019-0008` in #117

**Pull request merged**